web-app/src/app/(landing)/(auth)/forgot-password/data.ts

```diff
@@ -3,11 +3,13 @@ import { z } from "zod";
 import { FormData, FormIntlTranslation } from "@/lib/form";
 
 const forgotPasswordFormSchema = (
-  t: FormIntlTranslation<"Auth.Pages.ForgotPassword.Form">,
+  t:
+    | FormIntlTranslation<"Auth.Pages.ForgotPassword.Form">
+    | undefined = undefined,
 ) =>
   z.object({
     email: z.string().email({
-      message: t("Errors.Email.invalid"),
+      message: t?.("Errors.Email.invalid"),
```

Comment:
Maybe we need a fallback message or require the FormIntl

Reply:
No, at the moment it's optional because we also use the same schema for validation on the server side. During this validation, a message is not needed.

Example Server https://github.com/masumi-network/sokosumi/pull/112/files#diff-6b817cac40b7014592a9851b726bbe4c5a86802e12311502dc6e8875ac5b2086R8